### DISCUSSION TITLE
프로젝트 README 템플릿 정의하기

### DISCUSSION THREAD

**Discussion by Cotidie** [ApptiveDev/.github#7] [Ideas]
프로젝트는 백엔드, 프론트엔드 리포지토리를 각각 서브모듈로 구성하고자 하고, 다음 세 가지의 README 템플릿을 정의한다면 좋겠습니다.

- 프로젝트 README
- 백엔드 README
- 프론트엔드 README

항목 구성은 대략 다음과 같이 생각하고 있습니다.  
README 구성과 각 항목의 가이드라인에 대해 아이디어를 내주세요.

- 개요
  - 프로젝트 기능, 해결하고자 하는 문제 등
- 구성원
- 구조도/아키텍처
- 배포 방법
- 사용 스택
- 스크린샷
- 라이센스

배포와 README까지 작성해야 프로젝트는 비로소 완성됩니다.

**Comment by ChoMinGi** [comment]
- 개요,구성원
- 구조도/아키텍쳐
- 배포방법

등에 대한 것은 프로젝트 README(상위)에 작성하고 나머지는 각각에 작성하는건 어떨까요?

구조도나 배포 아키텍쳐의 경우에는 figma에 aws 아이콘이 적절히 제공되고 있습니다.
